packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -64,6 +64,9 @@ export interface IPushOptions {
       };
     };
     fcmOptions?: { analyticsLabel?: string };
+    'one-signal'?: {
```

Comment:
Either this or the `ITriggerOverrides` in `events.interface` is wrong.
We should add tests to the provider to double check the right implementation.

Reply:
Thanks for the feedback. Forgot about that one. Will try to fix and include the docs tomorrow